docs/learning/index.md

```diff
@@ -1,18 +1,14 @@
-# Documentation Overview
+# Getting started with iTwin.js
 
-This website is organized into sections by tabs at the top of each page.
+iTwin.js has a comprehensive set of APIs that can be used anywhere an [iModel](./imodels.md) may be relevant. It's a big library: where you start will depend on what you'd like to build. If you're looking to extend an iTwin viewer's user interface -- perhaps adding a new tool or widget -- [check out the frontend guide](./frontend/index). For higher level React components, you may want to [take a look at AppUI](./ui/appui/index.md).
```

Comment:
i think we might want to word smith this a bit further

Reply:
I've updated the text for your consideraation.